frontends/web/src/routes/buy/exchange.tsx

```diff
@@ -116,8 +121,8 @@ export const Exchange = ({ code, accounts }: TProps) => {
     if (!exchangeDeals) {
       return;
     }
-    const pocketBest = findBestDeal(exchangeDeals.pocket);
-    const moonpayBest = findBestDeal(exchangeDeals.moonpay);
+    const pocketBest = findBestDeal(exchangeDeals.exchanges[0].deals);
```

Comment:
This is horrible but is just a draft, meant to be replaced by @shonsirsha code, rebasing https://github.com/digitalbitbox/bitbox-wallet-app/pull/1881#
I didn't want to change the code too much here, to avoid complicating rebase conflicts.